shared-module/displayio/Palette.c

```diff
@@ -89,6 +89,9 @@ bool displayio_palette_get_color(displayio_palette_t *self, const _displayio_col
     } else if (colorspace->grayscale) {
         size_t bitmask = (1 << colorspace->depth) - 1;
         *color = (self->colors[palette_index].luma >> colorspace->grayscale_bit) & bitmask;
+    } else if (colorspace->depth == 32) {
```

Comment:
This was set to 32 bits (as opposed to 24) as in other place some division took place and 24 would require more changes. Handling 24 bit color in 32 bits is done in other systems but not sure if it is the best option here or not?

Reply:
I think I added this as 32 bits for the raspberry pi too. In that case, the last byte is actually alpha. So, this works well for me :-).